run/cifs_archive/connect-archive.sh

```diff
@@ -1,28 +1,9 @@
 #!/bin/bash -eu
 
-function ensure_archive_is_mounted () {
-  if [ -e "$ARCHIVE_MOUNT" ]
-  then
-    log "Ensuring cam archive is mounted..."
-    if ensure_mountpoint_is_mounted_with_retry "$ARCHIVE_MOUNT"
-    then
-      log "Ensured cam archive is mounted."
-    else
-      log "Failed to mount cam archive."
-      return 1
-    fi
-  fi
-  if [ -e "$MUSIC_ARCHIVE_MOUNT" ]
-  then
-    log "Ensuring music archive is mounted..."
-    if ensure_mountpoint_is_mounted_with_retry "$MUSIC_ARCHIVE_MOUNT"
-    then
-      log "Ensured music archive is mounted."
-    else
-      log "Failed to mount music archive."
-      return 1
-    fi
-  fi
+function mount_if_set() {
+  local mount_point=$1
+  [ -n "$mount_point" ] && ensure_mountpoint_is_mounted_with_retry "$mount_point"
```

Comment:
I think this should be `[ -z "$mount_point" ] || ensure_mountpoint_is_mounted_with_retry "$mount_point"` so that it doesn't fail when